model/qti/ImportService.php

```diff
@@ -112,12 +112,11 @@ public function importQTIFile($qtiFile, core_kernel_classes_Class $itemClass, $v
      *
      * @param core_kernel_classes_Class $itemClass
      * @param Item $qtiModel
-     * @param Resource $qtiItemResource
      * @throws common_exception_Error
      * @throws \common_Exception
      * @return core_kernel_classes_Resource
      */
-    protected function createRdfItem(core_kernel_classes_Class $itemClass, Item $qtiModel, Resource $qtiItemResource)
+    protected function createRdfItem(core_kernel_classes_Class $itemClass, Item $qtiModel)
```

Comment:
Is this change related to the feature ?
I feel uncomfortable with that as I have no idea of the possible impacts...

Reply:
This is fixing and issue when you try to import a single xml file you get 500 error message.
I have other PR to fix this, but I have to merge that branch into this one to be able to test it.